ErlangInstallerPreferences/MainTabViewController.swift

```diff
@@ -68,27 +68,27 @@ class MainTabViewController: NSViewController, NSTextFieldDelegate
     func scheduleCheckNewReleasesForAgent() {
         self.erlangInstallerApp?.checkNewReleases!()
     }
-
+    
     func loadVersionAndBuildNumber() {
         let version = Bundle.main.infoDictionary!["CFBundleShortVersionString"] as? String
         let build = Bundle.main.object(forInfoDictionaryKey: kCFBundleVersionKey as String) as? String
         self.versionAndBuildNumber.stringValue = "Version " + version! + " Build " + build!
     }
-
-    @IBAction func openAtLogin(_ sender: AnyObject) {
     
+    @IBAction func openAtLogin(_ sender: AnyObject) {
+        
         // FIXME: sub project not necessary when this is fixed.
         //			if !SMLoginItemSetEnabled(("com.erlang-solutions.ErlangInstaller-Helper" as CFString), Bool(sender.state)) {
         //				print("Setting as login item was not successful")
         //			}
-        UserDefaults.openAtLogin = self.openAtLogin.state == 1
-        let url = NSWorkspace.shared().urlForApplication(withBundleIdentifier: Constants.applicationId)
+        UserDefaults.openAtLogin = self.openAtLogin.state.rawValue == 1
+        let url = NSWorkspace.shared.urlForApplication(withBundleIdentifier: Constants.applicationId)
         _ = Utils.setLaunchAtLogin(url!, enabled: UserDefaults.openAtLogin)
     }
     
     @IBAction func checkNewReleasesClick(_ sender: AnyObject) {
-    
-        UserDefaults.checkForNewReleases = self.checkForNewReleases.state == 1
+        
+        UserDefaults.checkForNewReleases = self.checkForNewReleases.state.rawValue == 1
```

Comment:
Again, just for my own curiosity, what does 'rawValue' signify ?

Reply:
The underlying integer on an enum.